common/it/com/commercetools/sunrise/common/WithSphereClient.java

```diff
@@ -44,7 +44,7 @@ protected static String clientSecret() {
         return getValueForEnvVar(IT_CTP_CLIENT_SECRET);
     }
 
-    private synchronized static BlockingSphereClient sphereClient() {
+    protected synchronized static BlockingSphereClient sphereClient() {
```

Comment:
No, I would not reuse this from the JVM SDK because there is too much surprises hidden.